main.go

```diff
@@ -106,14 +122,15 @@ func main() {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
-
-	if err = (&controllers.DatabaseClusterReconciler{
+	dbReconciler := &controllers.DatabaseClusterReconciler{
 		Client: mgr.GetClient(),
 		Scheme: mgr.GetScheme(),
-	}).SetupWithManager(mgr); err != nil {
+	}
+	if err := dbReconciler.SetupWithManager(mgr); err != nil {
```

Comment:
Yep, I know that the controller for BackupStorage is missing to run the reconciliation loop either for backupstorage update or for a secret update